static/js/settings_account.js

```diff
@@ -22,6 +22,42 @@ function settings_change_success(message) {
 exports.set_up = function () {
     $("#account-settings-status").hide();
 
+    var change_email_modal = new Modal(
+        templates.render('change_email_modal', { email: page_params.email })
+    );
+
+    change_email_modal.setUp(function (mounted_modal) {
```

Comment:
What in particular do you find confusing -- any suggestions for an alternative? Maybe just `modal` would be fine.